packages/mongo/mongo_driver.js

```diff
@@ -1039,9 +1039,14 @@ class AsynchronousCursor {
 
     this._visitedIds = new LocalCollection._IdMap;
   }
-
-  [Symbol.iterator]() {
-    return this._cursor[Symbol.iterator]();
+  
+  [Symbol.asyncIterator]() {
+    return {
+      async next {
```

Comment:
Sorry, I missed the parenthesis here 🙈 
```suggestion
      async next() {
```

Reply:
I didn't notice either 🤷